### ISSUE TITLE
Regarding documentation for RegNets

### ISSUE THREAD

**Issue by AdityaKane2001** [keras-team/keras#15895]
@fchollet

[#15702](https://github.com/keras-team/keras/pull/15702) was merged recently in dd73626. I wanted to add the performance metrics to the applications table in keras.io. However, I am not able to spin up a VM of the given specifications[^1]  [^2]. 

Could you please clarify how to go about this?

/cc @sayakpaul 
[^1]: CPU: AMD EPYC Processor (with IBPB) (92 core) - Ram: 1.7T - GPU: Tesla A100 
[^2]: [Comment](https://github.com/keras-team/keras/pull/15702#issuecomment-995409740)

**Comment by qlzh727** [comment]
Triage notes: We should provide a user accessible way to produce performance benchmarks. Or we need to at least have it available internally to produce the number for user.

**Comment by sayakpaul** [comment]
I think it might be useful to put up a guide for benchmarks for contributors.

**Comment by qlzh727** [comment]
Thanks @sayakpaul. Please feel free to write up a guide for how to setup this env. 

On a side note, we are going to explore a setup for keras cv and nlp to benchmark the performance for applications.

**Comment by sayakpaul** [comment]
I think the Keras team should write to reduce the chances of errors, inconsistencies, etc. 

But if you could share some references on how the team usually goes about it, I can take a dig. 

For benchmarking performance (not the throughput, just accuracy), I usually do something like this:

https://github.com/sayakpaul/ViT-jax2tf/tree/main/i1k_eval.
